mmcls/models/classifiers/image.py

```diff
@@ -7,16 +7,54 @@
 
 @CLASSIFIERS.register_module()
 class ImageClassifier(BaseClassifier):
+    arch_zoo = {
```

Comment:
Thank you for your PR, You reminded us that we should have a simpler model interface

We have discussed the design of this PR, which is equivalent to copying the content of 'configs/_base_/models' here, which will be very lengthy.

We plan to use the (name: config) pair in "./configs/xxxxnet/metafile.yml" after 1.x release, such as https://github.com/open-mmlab/mmclassification/blob/90254a845540650bd03bc2b6108251825935c687/configs/conformer/ metafile.yml#L18 implements this kind of functionality.

Reply:
Sounds good, thank you.